### PR TITLE
added vite

### PR DESCRIPTION
**Pull request by samyabrata-maji**
## 👨‍💻 Changes proposed
- [x] added vite as dev dependency
- [x] added scripts to run vite servers
- [x] created main.jsx which is the entrypoint (if vite is used). For now main.jsx has a dummy text.
- [x] added path aliases for vite

### CODE REVIEW

**Comment by samyabrata-maji** [comment]
- [x] added path aliases for vite

**Comment by samyabrata-maji** [comment]
Fixed parsing error
Added `@babel/eslint-parser` in the `.eslintrc.json` file

parser
parser2

**Comment by samyabrata-maji** [comment]
1. Vite uses `import.meta.env` instead of `process.env` to access environment variables
2. Only the environment variables starting with `VITE_` are exposed.

File: vite.config.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+const path = require('path')
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  resolve: {
+    alias: [
+      {
+        find: 'actions',
+        replacement: path.resolve(__dirname, './src/actions'),
+      },
```
Comment (nilsnolde) [inline vite.config.js:14]: so it needs to be told how to resolve import paths? or what is that for?

Comment (samyabrata-maji) [reply vite.config.js:14]: It matches for the word in the import statement and replaces it with the given path.

so, vite interprets: `import "action/file.jsx"` as  `import "./src/actions/file.jsx"`

Its optional.

**Pull request closed without merging**